packages/indexer/src/handlers/events/evmHost/getRequestHandled.event.handler.ts

```diff
@@ -67,6 +91,10 @@ export const handleGetRequestHandledEvent = wrap(async (event: GetRequestHandled
 					blockTimestamp,
 				)
 				await VolumeService.updateVolume(`Transfer.${symbol}`, amountValueInUSD, blockTimestamp)
```

Comment:
Since we know multiple get responses can exist in a single transaction, wouldn't this count the same volume for every get response in a batch?